samples/frameworks/react-native-expo/ts/appconfigBasic/appConfigSample.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @summary Demonstrates the CRUD operations on the configuration settings.
+ */
+import { AppConfigurationClient } from "@azure/app-configuration";
+
+export async function runAppConfigSample() {
+  console.log(`Running helloworld sample`);
+
+  // Set the following environment variable or edit the value on the following line.
+  const connectionString =
+    process.env["APPCONFIG_CONNECTION_STRING"] || "<connection string>";
+  const client = new AppConfigurationClient(connectionString);
+
+  const greetingKey = "Samples:Greeting";
+
+  await cleanupSampleValues([greetingKey], client);
+
+  // creating a new setting
+  console.log(`Adding in new setting ${greetingKey}`);
```

Comment:
It's probably better to update the UI instead of console.log...

Reply:
Do you want to work on this as a separate PR?

Reply:
I was thinking about UI that allows user to specify key name and value but console.log is fine for now to show that our code runs on the platform.